docs/import/score_bounds_map.js

```diff
@@ -142,10 +142,12 @@ class ScoreBoundsMap {
 
 /**
  * Sets up `polygon` to call `callback` whenever the polygon changes. Normal
- * shape changes are handled by the listeners registered by {@link
- * addListenersToPolygon}, while drag events are handled separately. Since drag
- * events generate so many individual change events, we remove those events on
- * drag start and add them back on drag end.
+ * shape changes (dragging a corner, dragging the middle of an edge point) are
+ * handled by the listeners registered by {@link
+ * ScoreBoundsMap._addListenersToPolygon}, while moving the whole polygon (via
```

Comment:
Ah I didn't realize you could drag the whole thing! That's why I was confused. Dragging the whole polygon works but I also get this stack trace in the dev console:

score_bounds_map.js:161 Uncaught ReferenceError: addListenersToPolygon is not defined
    at _.zi.<anonymous> (score_bounds_map.js:161)
    at Be.j (js?libraries=drawing,places&key=AIzaSyBAQkh-kRrYitkPafxVLoZx3E5aYM-auXM:217)
    at Object._.O.trigger (js?libraries=drawing,places&key=AIzaSyBAQkh-kRrYitkPafxVLoZx3E5aYM-auXM:214)
    at EX (poly.js:23)
    at DX._.r.dj (poly.js:91)
    at Be.j (js?libraries=drawing,places&key=AIzaSyBAQkh-kRrYitkPafxVLoZx3E5aYM-auXM:217)
    at Object._.O.trigger (js?libraries=drawing,places&key=AIzaSyBAQkh-kRrYitkPafxVLoZx3E5aYM-auXM:214)
    at iY.handleEvent (poly.js:101)
    at Dv (map.js:23)
    at Av (map.js:22)

Reply:
Sorry, javascript refactoring isn't what it could be. Fixed, and added a test.